code/modules/projectiles/guns/projectile/automatic.dm

```diff
@@ -222,7 +222,7 @@
 	origin_tech = "combat=6;materials=1;syndicate=2"
 	slot_flags = SLOT_BACK
 	ammo_type = "/obj/item/ammo_casing/a762"
-	allowed_magazines = list(/obj/item/ammo_casing/a762)
+	allowed_magazines = list(/obj/item/ammo_magazine/a762)

```

Comment:
What is this change about? The PR text doesnt seem to mention anything about ammunition.